pages/index.tsx

```diff
@@ -13,8 +13,11 @@ export default function Home() {
       </Head>
 
       <main className={styles.main}>
-        <Logo size="big"></Logo>
-        <LoginButton></LoginButton>
+        <div>
+          <Logo size="big"></Logo>
+          <LoginButton></LoginButton>
+          <SkipLoginButton></SkipLoginButton>
+        </div>
```

Comment:
This div seems unnecessary here
```suggestion
          <Logo size="big"></Logo>
          <LoginButton></LoginButton>
          <SkipLoginButton></SkipLoginButton>
```